node/tests/TestUtilities.ts

```diff
@@ -40,6 +40,17 @@ export function flushallOnPort(port: number): Promise<void> {
     );
 }
 
+export const parseEndpoints = (endpoints: string): [string, number][] => {
+    if (!endpoints.trim()) {
```

Comment:
why? what does this scenario mean? if endpoints is an empty string, please add appropriate check